mne/decoding/time_gen.py

```diff
@@ -390,11 +446,13 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode,
     ----------
     X : ndarray, shape (n_epochs, n_features, n_times)
         To-be-fitted data.
-    estimators : array-like, shape (n_times, n_folds)
-        Array of scikit-learn classifiers fitted in cross-validation.
-    slices : list
-        List of slices selecting data from X from which is prediction is
-        generated.
+    estimators : list of array-like, shape (n_times, n_folds)
+        List of array of scikit-learn classifiers fitted in cross-validation.
```

Comment:
Basically I was thinking of something like this:

```py

def score_func(gat):
    return gat.score()

def iterate_gat(func, train_times, test_times, n_jobs, *args, **kwargs)
    my_func = Parallel(func)
    n_jobs = check_n_jobs(n_jobs)
    # do the parallel stuff here
    return result
```
maximum flexibility. Plot, score, whatever you want.